src/main/java/io/ncbpfluffybear/supserv/SupServItems.java

```diff
@@ -396,7 +394,7 @@ private static void addPressRecipe(int seconds, ItemStack input, ItemStack outpu
     /**
      * Registers a recipe to Ore Crusher
      *
-     * @param input an array of items that are inserted
+     * @param input  an array of items that are inserted
```

Comment:
Redundant space

Reply:
This was placed there by intellij code formatter, but i removed it.

Reply:
Also its the same way with previous javadoc, it is filled with spaces to align lines
![image](https://user-images.githubusercontent.com/40923017/157308335-079090a2-0668-4c14-af7e-8dee476c5224.png)